open_schools_platform/history_management/views.py

```diff
@@ -91,3 +96,63 @@ def get(self, request, pk):
                               empty_exception=True,
                               empty_message="There is no such student")
         return Response({"results": StudentHistorySerializer(student).data}, status=200)
+
+
+class StudentProfileHistoryApi(ApiAuthMixin, APIView):
+    @swagger_auto_schema(
+        operation_description="Get student-profile history",
+        tags=[SwaggerTags.HISTORY_MANAGEMENT],
+        responses={200: swagger_dict_response({'results': StudentProfileHistorySerializer()}),
+                   404: "There is no such student-profile"},
+    )
+    def get(self, request, pk):
+        student_profile = get_student_profile(filters={"id": pk},
+                                              user=request.user,
+                                              empty_exception=True,
+                                              empty_message="There is no such student-profile")
+        return Response({"results": StudentProfileHistorySerializer(student_profile).data}, status=200)
+
+
+class EmployeeProfileHistoryApi(ApiAuthMixin, APIView):
+    @swagger_auto_schema(
+        operation_description="Get employee-profile history",
+        tags=[SwaggerTags.HISTORY_MANAGEMENT],
+        responses={200: swagger_dict_response({'results': EmployeeProfileHistorySerializer()}),
+                   404: "There is no such employee-profile"},
+    )
+    def get(self, request, pk):
```

Comment:
We already know request.user employee_profile, so there is no need of passing it's pk to the handler. You need to refactor url too.